hardware/OTGWBase.cpp

```diff
@@ -400,8 +395,10 @@ void OTGWBase::ParseLine()
 		_status.Return_water_temperature = static_cast<float>(atof(results[idx++].c_str()));				SendTempSensor(idx - 1, 255, _status.Return_water_temperature, "Return Water Temperature");
 		_status.DHW_setpoint_boundaries=results[idx++];
 		_status.Max_CH_setpoint_boundaries=results[idx++];
-		_status.DHW_setpoint = static_cast<float>(atof(results[idx++].c_str())); if(_status.DHW_setpoint!=0.0f)							UpdateSetPointSensor(idx - 1, _status.DHW_setpoint, "DHW Setpoint");
-		_status.Max_CH_water_setpoint = static_cast<float>(atof(results[idx++].c_str()));					UpdateSetPointSensor(idx - 1, _status.Max_CH_water_setpoint, "Max_CH Water Setpoint");
+		_status.DHW_setpoint = static_cast<float>(atof(results[idx++].c_str())); 
+		if(_status.DHW_setpoint!=0.0f) UpdateSetPointSensor(idx - 1, _status.DHW_setpoint, "DHW Setpoint");
```

Comment:
Please follow the default coding standards
if (1==1)
<ident>do something

or 

if (1==1)
{
<ident>do something
}